tests/Integration/goldens/dataproc/src/V1/Client/BaseClient/AutoscalingPolicyServiceBaseClient.php

```diff
@@ -91,6 +93,130 @@ private static function getClientDefaults()
+     * @param string $location
+     * @param string $autoscalingPolicy
+     *
+     * @return string The formatted autoscaling_policy resource.
+     */
+    public static function autoscalingPolicyName($project, $location, $autoscalingPolicy)
+    {
+        return self::getPathTemplate('autoscalingPolicy')->render([
+            'project' => $project,
+            'location' => $location,
+            'autoscaling_policy' => $autoscalingPolicy,
+        ]);
+    }
+
+    /**
+     * Formats a string containing the fully-qualified path to represent a location
+     * resource.
+     *
+     * @param string $project
+     * @param string $location
+     *
+     * @return string The formatted location resource.
+     */
+    public static function locationName($project, $location)
+    {
+        return self::getPathTemplate('location')->render([
+            'project' => $project,
+            'location' => $location,
+        ]);
+    }
+
+    /**
+     * Formats a string containing the fully-qualified path to represent a
+     * project_location_autoscaling_policy resource.
+     *
+     * @param string $project
+     * @param string $location
+     * @param string $autoscalingPolicy
+     *
+     * @return string The formatted project_location_autoscaling_policy resource.
+     */
+    public static function projectLocationAutoscalingPolicyName($project, $location, $autoscalingPolicy)
+    {
+        return self::getPathTemplate('projectLocationAutoscalingPolicy')->render([
+            'project' => $project,
+            'location' => $location,
+            'autoscaling_policy' => $autoscalingPolicy,
+        ]);
+    }
+
+    /**
+     * Formats a string containing the fully-qualified path to represent a
+     * project_region_autoscaling_policy resource.
+     *
+     * @param string $project
+     * @param string $region
+     * @param string $autoscalingPolicy
+     *
+     * @return string The formatted project_region_autoscaling_policy resource.
+     */
+    public static function projectRegionAutoscalingPolicyName($project, $region, $autoscalingPolicy)
+    {
+        return self::getPathTemplate('projectRegionAutoscalingPolicy')->render([
+            'project' => $project,
+            'region' => $region,
+            'autoscaling_policy' => $autoscalingPolicy,
+        ]);
+    }
+
+    /**
+     * Formats a string containing the fully-qualified path to represent a region
+     * resource.
+     *
+     * @param string $project
+     * @param string $region
+     *
+     * @return string The formatted region resource.
+     */
+    public static function regionName($project, $region)
+    {
+        return self::getPathTemplate('region')->render([
+            'project' => $project,
+            'region' => $region,
+        ]);
+    }
+
+    private static function registerPathTemplates()
+    {
+        self::loadPathTemplates(__DIR__ . '/../../resources/autoscaling_policy_service_descriptor_config.php', self::SERVICE_NAME);
```

Comment:
nit but can we break this up?
```suggestion
        self::loadPathTemplates(
            __DIR__ . '/../../resources/autoscaling_policy_service_descriptor_config.php',
            self::SERVICE_NAME
        );
```

We should just need to take advantage of the [Formatter:: buildLineLengthFixer](https://github.com/googleapis/gapic-generator-php/blob/main/src/Utils/Formatter.php#L166) method by passing in `$lineLength`.

Reply:
Oops i shouldn't have committed that haha I will do as you suggested but in the generator code.

Reply:
What kind of `$lineLength` do you want? I will have to use it for the entire v2 GAPIC client file.

Reply:
To get this to reliably wrap, we probably need to target `80`

Reply:
When adding the V2 clients I tried to add a line length formatter, but it did some odd formatting I wasn't super happy with. I'd like to do a follow up PR to address, so I wouldn't worry too much about the formatting right now.

Reply:
Thanks I think that makes sense...trying `80` did make some weird things...

Reply:
We should look at doing more localized formatting in the `AST::block` code gen  maybe, for things like this